graphql_client_cli/src/generate.rs

```diff
@@ -21,10 +23,18 @@ pub fn generate_code(
         _ => None,
     };
 
+    let module_visibility = module_visibility.as_ref().map(|s| s.as_str());
+    let module_visibility = match module_visibility {
+        Some("pub") => syn::VisPublic { pub_token: <Token![pub]>::default() }.into(),
+        Some("private") => syn::Visibility::Inherited {},
+        _ => syn::VisPublic { pub_token: <Token![pub]>::default() }.into(),
```

Comment:
`Restricted(VisRestricted)` needs complex variables.
So I implement pub and private at first for cli.